content/blog/pdf-qa-application-with-bedrock/index.md

```diff
@@ -139,12 +139,13 @@ from haystack_integrations.components.retrievers.opensearch import OpenSearchBM2
 ## Create pipeline components
 retriever = OpenSearchBM25Retriever(document_store=document_store, top_k=15)
 
+## Set the AWS credentials as environment variables
+os.environ["AWS_ACCESS_KEY_ID"] = "aws_access_key_id"
+os.environ["AWS_SECRET_ACCESS_KEY"] = "aws_secret"
+os.environ["AWS_REGION_NAME"] = "aws_region_name"
```

Comment:
I'm asking this to make sure: do I need to set `AWS_REGION_NAME` as an environment variable or is it a regular string that I can pass to the component at init without using `Secret` methods?

Reply:
with the `Secret` component regular strings are no longer accepted